### PR TITLE
add `noexec` error to FAQ

### LINKED ISSUE #1370: noexec on mounted filesystem error
Should add an FAQ for this issue:

I couldn't run `/bin/sh` within a container because it was mounted with noexec option. 

Issue found with: `df -h /bin/sh` to get mounted filesystem and then `cat /proc/self/mountinfo | grep MOUNT_LOCATION` to see if nonexec option is set.

Fixed with: `mount -o remount, exec MOUNT_LOCATION`

### PR DESCRIPTION
**Pull request by lucaudill**
closes #1370

### CODE REVIEW

**Review by reidpr — COMMENTED** [review]
Nice work. Please check my tidy. If it looks OK, have @mphinney1100 review it for clarity.

**Comment by lucaudill** [comment]
Looks good to me. Requesting a review from @mphinney1100.

File: doc/faq.rst [hunk 1]
```diff
@@ -195,6 +195,41 @@ system level, not the user level. In the example above, this means changing
...
+  └─/var/tmp  tmpfs   tmpfs   rw,noexec,relatime,size=8675309k
+
+Note :code:`noexec` under :code:`OPTIONS`.
+
+To fix this, you can:
+
+  1. Use a different filesystem mounted :code:`exec` (i.e., the opposite
+     of :code:`noexec` and typically the default).
+
+  2. Change the mount options for the filesystem.
```
Comment (mphinney1100) [inline doc/faq.rst:227]: I would reword to to say: "Remount with exec options for the filesystem" or something like that.

**Review by mphinney1100 — APPROVED** [review]
I added 1 comment but looks good.
